alibi_detect/saving/tensorflow/_loading.py

```diff
@@ -274,7 +274,7 @@ def load_detector_legacy(filepath: Union[str, os.PathLike], suffix: str, **kwarg
             emb, tokenizer = load_text_embed(filepath)
         try:  # legacy load_model behaviour was to return None if not found. Now it raises error, hence need try-except.
             model = load_model(filepath, load_dir='encoder')
-        except FileNotFoundError:
+        except OSError:
```

Comment:
Changed to `OSError` because we are now relying on `tf.keras.model.load_model` to raise an error when loading fails, and this is what it raises...